vue-app/src/views/VerifyLanding.vue

```diff
@@ -85,7 +93,7 @@ export default class VerifyLanding extends Vue {
 
   get isRoundFullOrOver(): boolean {
     return (
-      this.$store.getters.isRecipientRegistryFull ||
```

Comment:
Don't we also want to check if the recipient registry is full? That can happen without hitting the message cap.

Reply:
Ah excuse me - I meant this should also check for if `maxContributors` has been reached.